plugins/CoreUpdater/templates/layout.twig

```diff
@@ -42,13 +42,17 @@
 <body id="simple" ng-app="app" class="old-ie"> <![endif]-->
 <!--[if (gte IE 9)|!(IE)]><!-->
 <body id="simple" ng-app="app"><!--<![endif]-->
-<div id="contentsimple">
-    <div id="title">
-        <img title='Piwik' alt="Piwik" src="plugins/Morpheus/images/logo-header.png" style="margin-left:10px;"/>
-        <span id="subh1"> # {{ 'General_OpenSourceWebAnalytics'|translate }}</span>
-    </div>
+
+<div class="logo">
+    <img title="Piwik" alt="Piwik" src="plugins/Morpheus/images/logo-header.png"/>
```

Comment:
This is the diff for #7578 (updater redesign).

I've added support for custom logo in the maintenance mode, I'll try to add support for that in the updater in #7578.